rental_quality_control/models/sale_rental.py

```diff
@@ -0,0 +1,49 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleRental(models.Model):
+    _inherit = "sale.rental"
+
+    qc_inspections_ids = fields.One2many(
+        "qc.inspection",
+        "rental_id",
+        string="Inspections",
+    )
+
+    created_inspections = fields.Integer(compute="_compute_qc_counts")
+    done_inspections = fields.Integer(compute="_compute_qc_counts")
+    passed_inspections = fields.Integer(
+        compute="_compute_qc_counts", string="Inspections OK"
+    )
+    failed_inspections = fields.Integer(
+        compute="_compute_qc_counts", string="Inspections failed"
+    )
+
+    @api.depends("qc_inspections_ids.state")
+    def _compute_qc_counts(self):
+        data = (
+            self.env["qc.inspection"]
+            .sudo()
+            .read_group(
```

Comment:
```suggestion
            self.env["qc.inspection"]
            .sudo()
            .read_group(
```
there are two things I'd like to clarify:

do we really need to use `sudo()` here?
we should respect the normal access rules first and only escalate privileges when absolutely necessary.

shouldn't using read_group instead of _read_group be encouraged on the backend?
there is an ongoing discussion about this in here: `https:⧸⧸github.com/odoo/odoo/pull/136381`

Reply:
Thanks, I fixed it

Reply:
@xaviedoanhduy you mean `_read_group` instead of `read_group` right?

Reply:
yeah sorry for my copy and paste is bad